src/types/globalTypes.ts

```diff
@@ -1342,6 +1347,24 @@ export interface OrderLineInput {
   quantity: number;
 }
 
+export interface OrderRefundFulfillmentLineInput {
+  fulfillmentLineId?: string | null;
+  quantity: number;
+}
+
+export interface OrderRefundLineInput {
+  orderLineId?: string | null;
+  quantity: number;
+}
+
+export interface OrderRefundProductsInput {
+  orderLines?: OrderRefundLineInput[] | null;
+  fulfillmentLines?: OrderRefundFulfillmentLineInput[] | null;
+  notifyCustomer?: boolean | null;
+  amountToRefund?: any | null;
```

Comment:
can we type this more strictly?

Reply:
we can't, these types are automatically generated from graphql schema

Reply:
at least without changing the way in which it is generated. There are many similar cases in this global types.